qemu/deps/playbook.yml

```diff
@@ -40,18 +39,39 @@
         /usr/bin/update-ca-trust extract
       when: dnld_cert_output.changed
 
-# The following two tasks need to be modified, check if kar already deployed, if true, skip.
-    - name: clone kar
+    - name: Check kar folder exists
+      stat:
+         path: '{{kar_local}}'
+      register: kar_folder
+
+    - name: Clone kar
       git:
         repo: '{{kar_repo}}'
         dest: '{{kar_local}}'
         force: yes
+      when: not kar_folder.stat.exists
 
     - name: Run bootstrap script
       command: ./Bootstrap.sh {{bs_options}}
       args:
         chdir: '{{kar_local}}'
+      when: not kar_folder.stat.exists
       register: bootstrap_result
+      ignore_errors: True
+
+    - name: Remove kar folder
```

Comment:
`Remove kar folder when bootstrap failed `  ?

Reply:
Yes, if bootstrap failed, remove kar folder and uninstall rpms.

Reply:
@qcheng-redhat I mean update the comment for this task, to make it more clear.

Reply:
Updated task name.